src/main/fc/config.c

```diff
@@ -631,6 +631,10 @@ void validateAndFixConfig(void)
     validateAndFixGyroConfig();
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
+    featureSet(FEATURE_RUNCAM_SPLIT);
```

Comment:
featureSet(FEATURE_RUNCAM_SPLIT);
Yes, runcam split is not a common device, the code may not be appropriate.
But this code is used to determine whether or not to display RunCam Split Tab on CleanFlight Configurator, user need goto the RunCam Split tab to setting some control setting.
When the CleanFlight Configurator got this feature(FEATURE_RUNCAM_SPLIT), it's will display RunCam Split, 

I'm new to cleanflight, Is there  have a better way to do this? Thank you!